config/_default/config.yml

```diff
@@ -13,6 +13,18 @@ module:
           target: static
         - source: layouts
           target: layouts
+    - path: github.com/pulumi/registry/themes/default
```

Comment:
Will this cause a problem for the docs repo during a full site build? I ask because registry also this repo as a dep and both of these modules join during the full site build. If you've verified this won't cause a problem, then please ignore this comment.

Reply:
Nope -- these Hugo `config` files only apply where the `hugo` command is run, so in this case, only in local and PR builds of this repo. Module consumers like pulumi/docs will never see them.